RevenueCatUI/CustomerCenter/ViewModels/BaseManageSubscriptionViewModel.swift

```diff
@@ -181,7 +181,10 @@ private extension BaseManageSubscriptionViewModel {
                 self.inAppBrowserURL = IdentifiableURL(url: url)
             }
 
-        case .changePlans, .cancel:
+        case .changePlans:
+            self.actionWrapper.handleAction(.showingChangePlans(purchaseInformation?.subscriptionGroupID ?? ""))
```

Comment:
would it be better to pass null instead of empty?

Reply:
Agreed. Let's work with `String?` instead